.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ssht"]
```

Comment:
Following the discussion on slack, we might just be able to change the url here.

Reply:
Yeah, I'll try that.